src/cli.ts

```diff
@@ -72,7 +73,7 @@ let p: Promise<void | {}>;
 
 switch (cmd) {
   case 'sync-repo-settings':
-    p = syncRepoSettings(cli.flags as Flags);
+    p = syncRepoSettings((cli.flags as unknown) as Flags);
```

Comment:
I prefer ignore linter rather than unknow

Reply:
Unfortunately this is not linter issue, but type incompatibility, it fails `npm test`. same as below.  :(

Reply:
Oh, I see. Thanks for explaining.